lib/ziti-tunnel-cbs/tests/all_tests.cpp

```diff
@@ -16,11 +16,11 @@ limitations under the License.
 
 // tweaks for mips-openwrt build
 // not sure if there is a better way
-#if defined(mips) || defined(__mips)
-#define nextafterl(x, y) nextafter(x,y)
-#define CATCH_CONFIG_NO_CPP11_TO_STRING
-#define CATCH_CONFIG_GLOBAL_NEXTAFTER
-#endif
+//#if defined(mips) || defined(__mips)
```

Comment:
Are these no longer needed (maybe due to C11 standard)? Any reason not to just remove?